lib/src/modules/inventoryItem/InventoryItem.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import cx from 'classnames';
+import { bool, string } from 'prop-types';
+import { ButtonIconContained, TooltipWrapper } from 'lib';
+
+export function InventoryItem({
+  isDragging,
+  isDragPreview,
+  iconName,
+  className,
+  tooltipClassName,
+  tooltipText,
+  ...rest
+}) {
+  const classNames = cx(`inventory-item ${className}`, {
+    'inventory-item-dragging': isDragging,
+    'inventory-item-preview': isDragPreview
+  });
+
+  if (isDragging) {
+    return <div className={classNames} />;
+  }
+
+  return (
+    <TooltipWrapper
+      id={`inventory-tooltip-${iconName}`}
```

Comment:
🟢  Is there not a chance that we'll end up with 2 ids that are the same? Does this matter?